content/ch-applications/hhl_tutorial.ipynb

```diff
@@ -999,13 +999,17 @@
    "source": [
     "from qiskit import ClassicalRegister, IBMQ\n",
     "from qiskit.compiler import transpile\n",
-    "from qiskit.ignis.mitigation.measurement import (complete_meas_cal, # Measurement error mitigation functions\n",
-    "                                                 CompleteMeasFitter, \n",
-    "                                                 MeasurementFilter)\n",
+    "#from qiskit.ignis.mitigation.measurement import (complete_meas_cal, CompleteMeasFitter, MeasurementFilter) # Measurement error mitigation functions\n",
```

Comment:
you can remove the line, no need to comment it out.

Reply:
okey